mods/d2k/rules/vehicles.yaml

```diff
@@ -337,6 +337,11 @@ devastator:
 		Step: 5
 		Ticks: 3
 		HealIfBelow: 50%
+	AttackDetonates:
```

Comment:
Upgradeable attack traits could solve this, but that feels like a big change in itself.